tests/runner.py

```diff
@@ -35,6 +35,9 @@ def path_from_root(*pathelems):
 except:
   raise Exception('Cannot find "COMPILER_OPTS" definition. Is %s set up properly? You may need to copy the template settings file into it.' % EM_CONFIG)
 
+# Force IDL checks
+os.environ['IDL_CHECKS'] = 'ALL'
```

Comment:
Agreed, though I'm not sure how I'd run the tests with different check modes, even with command line args. should I just recompile and re-run the tests 3 times?